cmd/kyverno/main.go

```diff
@@ -92,7 +92,7 @@ func main() {
 	flag.IntVar(&clientRateLimitBurst, "clientRateLimitBurst", 0, "Configure the maximum burst for throttle. Uses the client default if zero.")
 	flag.Func(toggle.AutogenInternalsFlagName, toggle.AutogenInternalsDescription, toggle.AutogenInternalsFlag)
 	flag.DurationVar(&webhookRegistrationTimeout, "webhookRegistrationTimeout", 120*time.Second, "Timeout for webhook registration, e.g., 30s, 1m, 5m.")
-	flag.IntVar(&changeRequestLimit, "changeRequestLimit", 100, "Configure the maximum count for report change requests.")
+	flag.IntVar(&changeRequestLimit, "changeRequestLimit", 1000, "Configure the maximum count for report change requests.")
```

Comment:
Is this per namespace? 1000 pending RCRs per namespace seems like a lot....do we have any test data to help determine a sensible default?

Reply:
Yes, this is per namespace.

Reply:
Per the reported issues, `resourceExhausted` occurs when there are more than ~2k results. So I set the limit to its half.

Reply:
In addition, it is not easy to completely opt-in via a feature flag, setting the limit to a large number keeps the reporting work as it was unless users set it to a small threshold.

Reply:
ok